node/core/approval-voting/src/lib.rs

```diff
@@ -2277,6 +2277,7 @@ async fn launch_approval(
 					available_data.pov,
 					APPROVAL_EXECUTION_TIMEOUT,
 					val_tx,
+					candidate.commitments_hash,
```

Comment:
With `CandidateReceipt` we already have a struct combining both the descriptor and the commitments ... why not just pass that, instead of both items separately?